packages/fxa-content-server/app/scripts/templates/post_verify/account_recovery/confirm_password.mustache

```diff
@@ -0,0 +1,35 @@
+<div id="main-content" class="card confirm-password">
+    <header>
+        <h1 id="fxa-confirm-password-header">
+            {{#serviceName}}
+                <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
+                {{#t}}Confirm password{{/t}} <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+            {{/serviceName}}
+            {{^serviceName}}
+                {{#t}}Confirm password{{/t}}
+            {{/serviceName}}
+        </h1>
+    </header>
+
+    <section>
+        <div class="error"></div>
+        <div class="success"></div>
+
+        <div class="graphic graphic-recovery-codes"></div>
+
+        <p class="verification-email-message">{{#unsafeTranslate}}Please confirm your password for %(escapedEmail)s to generate your recovery key.{{/unsafeTranslate}}</p>
+        <form novalidate>
+            <input type="email" class="email hidden" value="{{email}}" disabled />
+            <div class="input-row password-row">
+                <input type="password" class="password tooltip-below" id="password" placeholder="{{#t}}Enter Password{{/t}}" required pattern=".{8,}" autofocus/>
+            </div>
+
+            <div class="button-row">
+                <button id="submit-btn" type="submit">{{#t}}Continue{{/t}}</button>
+            </div>
+            <div class="links">
+                <a id="maybe-later-btn" data-flow-event="link.maybe-later">{{#t}}Maybe later{{/t}}</a>
+            </div>
+        </form>
+    </section>
+</div>
```

Comment:
We may want a follow-up issue to add an "eye" toggle to this input, but probably `needs:ux`.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/13018240/72829813-1f137400-3c45-11ea-8e18-6b00fdaa6dd5.png">

Reply:
Yea we should do that, although I thought this would have happened automatically since this was a password field 🤔